### ISSUE TITLE
[Pro] Unable to load the icon: 'UnityEditor.LookDevView'

### ISSUE THREAD

**Issue by LarsJandel** [KybernetikGames/inspector-gadgets#1]
Using Unity `2019.3.0f6`.
Using the feature `Persist After Play Mode`, the following error occurs and the `Watcher` doesn't `Auto Close`, which I guess it should, having the check mark next to it. It still works though, the values persist when exiting play mode. 🙂
Thanks
```
Unable to load the icon: 'UnityEditor.LookDevView'.
Note that either full project path should be used (with extension) or just the icon name if the icon is located in the following location: 'Assets/Editor Default Resources/Icons/' (without extension, since png is assumed)
UnityEditor.EditorGUIUtility:IconContent(String)
InspectorGadgets.Editor.WatcherWindow:OnEnable() (at Assets/Plugins/Inspector Gadgets/Serialized Properties/WatcherWindow.cs:29)
UnityEditor.EditorWindow:GetWindow()
InspectorGadgets.Editor.WatcherWindow:Watch(Object[]) (at Assets/Plugins/Inspector Gadgets/Serialized Properties/WatcherWindow.cs:329)
InspectorGadgets.Editor.PersistentValues:PersistObjects(Operation, Object[]) (at Assets/Plugins/Inspector Gadgets/Serialized Properties/PersistentValues.cs:84)
InspectorGadgets.Editor.<>c:<PersistComponent>b__5_0(List`1) (at Assets/Plugins/Inspector Gadgets/Serialized Properties/PersistentValues.cs:44)
InspectorGadgets.Editor.<>c__DisplayClass16_0:<GroupedInvoke>b__0() (at Assets/Plugins/Inspector Gadgets/Misc/IGEditorUtils.cs:223)
UnityEditor.EditorApplication:Internal_CallDelayFunctions()
```

**Comment by KybernetikGames** [comment]
Sorry I never responded about this, I didn't have notifications enabled for this repo.

I just released [Inspector Gadgets v6.2](https://kybernetik.com.au/inspector-gadgets/docs/changes/inspector-gadgets-v6) which fixes that issue.

**Comment by LarsJandel** [comment]
Thanks for the fix!
And no worries as it wasn't a show stopper. Otherwise I would have reached out on another channel. 😉
